tests/ext/otel_config_remapping.phpt

```diff
@@ -0,0 +1,40 @@
+--TEST--
+Test OpenTelemetry config remapping
+--ENV--
+OTEL_SERVICE_NAME=service
+OTEL_RESOURCE_ATTRIBUTES=foo=bar,deployment.environment=env,service.name=ignored,xyz=abc,service.version=1.2.3,baz=qux
+OTEL_TRACES_EXPORTER=none
+OTEL_METRICS_EXPORTER=none
+--INI--
+OTEL_SERVICE_NAME=other service
+OTEL_LOG_LEVEL=warn
```

Comment:
We have it the other way round: DD_TRACE_DEBUG=true sets DD_TRACE_LOG_LEVEL=debug. So it does not make much sense to do that here, except that it would confuse telemetry to distinguish the provenance of the configs.